vuepress/docs/v7.1/docs/reference/upgrade.md

```diff
@@ -0,0 +1,74 @@
+# Upgrade Guide E7.1.2 ⇒ E7.1.3
+
+## OLM
+
+Follow the steps below to upgrade your Entando OpenShift installation from 7.1.2 to 7.1.3.
```

Comment:
maybe move the version number closer to Entando, because the upgrade doesn't have anything to do with Openshift version